sdk/core/azure-core/src/main/java/com/azure/android/core/internal/util/test/EnqueueMockResponse.java

```diff
@@ -1,4 +1,4 @@
-package com.azure.android.core.http.interceptor;
+package com.azure.android.core.internal.util.test;
```

Comment:
I guess there are two things in the `test` project:

1. Unit tests for types in the `core` project.
2. Test types that we want to share with other "dependent" projects (e.g., storage).

For the second category, does it make sense to keeping these shared types rooted at `com.azure.android.core.test,` hence consistent with the established namespace in java [sdk](https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/core/azure-core-test/src/main/java/com/azure/core/test). Its a shippable package in java SDK, we haven't there yet for android but agreeing on the namespace will be a good start. Thoughts?

Reply:
Between why did we move "shared test" types from "src/test" to "src/main"?

Reply:
If discoverability of types are the reason for "src/test" -> "src/main" then worth checking this: https://stackoverflow.com/questions/5644011/multi-project-test-dependencies-with-gradle/60138176#60138176